scwx-qt/source/scwx/qt/util/qt6ct_palette.cpp

```diff
@@ -0,0 +1,87 @@
+/* This code is drawn from qt6ct, and loads qt6ct color palette files.
+ * qt6ct is licensed under BSD-2-Clause below.
+ *
```

Comment:
Prefer adding an explicit link to https://github.com/trialuser02/qt6ct/blob/master/src/qt6ct-common/qt6ct.cpp be included if not pulling in as a submodule. Makes it easier to trace back to the source if needed.